src/scripts/main.js

```diff
@@ -1,7 +1,39 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const createNotificationElement = () => {
+    const notification = document.createElement('div');
+
+    notification.className = `notification ${type}`;
+    notification.style.position = 'fixed';
+    notification.style.top = `${posTop}px`;
+    notification.style.right = `${posRight}px`;
+
+    const titleElement = document.createElement('h2');
+
+    titleElement.className = 'title';
+    titleElement.textContent = title;
+    notification.appendChild(titleElement);
+
+    const descriptionElement = document.createElement('p');
+
+    descriptionElement.textContent = description;
+    notification.appendChild(descriptionElement);
+
+    return notification;
+  };
+
+  const showNotification = (element) => {
+    document.body.appendChild(element);
+
+    setTimeout(() => {
+      document.body.removeChild(element);
```

Comment:
You're using `setTimeout` to remove the notification after 2 seconds. This is a magic number. It's better to move it to a constant at the beginning of your script, so it's clear what it represents and it can be easily changed.